gpkit/tests/t_model.py

```diff
@@ -42,6 +42,16 @@ def test_trivial_gp(self):
                                self.ndig)
         self.assertAlmostEqual(sol["cost"], 2*math.sqrt(2), self.ndig)
 
+    def test_601(self):
+        # tautological monomials should solve but not pass to the solver
+        x = Variable("x")
+        y = Variable("y", 2)
+        m = Model(x,
+                  [x >= 1,
+                   y == 2])
+        m.solve(verbosity=0)
+        self.assertEqual(len(m.program.constraints), 2)
```

Comment:
the length 2 assertion seems to conflict with the inline comment above -- if "tautological monomials should solve but not pass to the solver", then shouldn't the length of `m.program.constraints` be 1?